lib/rules/consistent-return.js

```diff
@@ -53,6 +53,8 @@ function isClassConstructor(node) {
 
 module.exports = {
     meta: {
+        type: "suggestion",
```

Comment:
I think this could also be `problem` since the goal is to catch forgotten `return` statements, but I don't feel strongly about it.

Reply:
The goal is really just to have consistency that `return` either always returns a value or doesn't, it doesn't actually flag an error. Consistency seems like it would be a suggestion.